backend/modules/vector_db/__init__.py

```diff
@@ -7,6 +9,7 @@
 
 SUPPORTED_VECTOR_DBS = {
     "qdrant": QdrantVectorDB,
+    # "mongo": MongoVectorDB,
```

Comment:
Is this intentional?

Reply:
Yes, Qdrant will always be used by default since its not dependent on creating an external account.